frontend/packages/core/src/card.tsx

```diff
@@ -11,21 +11,14 @@ import {
 import type { SpacingProps as MuiSpacingProps } from "@material-ui/system";
 import { spacing } from "@material-ui/system";
 
+import { IconButton } from "./button";
 import { Typography, TypographyProps } from "./typography";
 
 // TODO: seperate out the different card parts into various files
 
 const StyledCard = styled(MuiCard)({
   boxShadow: "0px 4px 6px rgba(53, 72, 212, 0.2)",
   border: "1px solid rgba(13, 16, 48, 0.1)",
-
-  ".MuiCardActionArea-root:hover": {
-    backgroundColor: "#F5F6FD",
-  },
-
-  ".MuiCardActionArea-root:active": {
```

Comment:
decided to style the CardActionArea itself since we're resuing these hover/active states in the "see more" action.